submission/submission-webapp/src/itest/resources/itestsInventory/itestsInventory.md

```diff
@@ -1,146 +1,147 @@
-| Class                           | Test No | Test name                                                             | Description                                                                                                     |
-|---------------------------------|---------|-----------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
```

Comment:
I think something went wrong with this file, why do we have so many changes? 🤔

Reply:
because my description column take longer than other so column have to be moved for everyrow